service/src/tests/parentchain_block_syncer_test.rs

```diff
@@ -0,0 +1,42 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use crate::{
+	tests::mocks::{
+		parentchain_api_mock::ParentchainApiMock, sidechain_api_mock::SidechainApiMock,
+	},
+	ParentchainBlockSyncer,
+};
+use itp_api_client_extensions::ChainApi;
+use std::sync::Arc;
+
+#[test]
+fn test_not_everything_explodes_when_gaudi_writes_a_test() {
+	let len = 42;
```

Comment:
Make your variable name more expressive - this is probably a `number of blocks` or something similar?